api/src/wfl/service/cromwell.clj

```diff
@@ -149,12 +149,14 @@
     (mapcat expand form-params)))
 
 (defn query
-  "Lazy results of querying Cromwell in ENVIRONMENT with PARAMS map."
+  "Lazy results of querying Cromwell in ENVIRONMENT with PARAMS.
+  If the params are a map they'll be passed to [[cromwellify-json-form]]."
```

Comment:
They can be a list, so that we can make our list structured in a way that a map doesn't allow (with different values for the same key). This is so we can get the multiple ID behavior:

```json
[
{"includeSubworkflows": "false"},
{"id": "a56fd1e5-fde8-42f2-8897-af64c1077878"},
{"id": "5a617a38-7dfd-40c3-b777-38339fd8e961"}
]
```